source/includes/administration/_commitments.md

```diff
@@ -227,3 +226,168 @@ Attributes | &nbsp;
+   "type": "RESOURCE",
+   "fixedPrice": 500.0,
+   "organization": {
+      "id": "e2c00fdb-d493-422c-889f-eb734806b80e"
+   },
+   "name": "fixed monthly price commitment",
+   "startDate": "2022-06-01T00:00:00Z"
+}
+```
+      
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+   "data": {
+      "committedProducts": [{
+         "createdAt": "2022-06-29T21:03:36.731Z",
+         "product": {
+            "id": "73ebf9c8-ecf7-4ee6-b881-150023d5f442",
+            "sku": "f"
+         },
+         "committedAmount": 1000.0,
+         "id": "6dc908c1-3e52-4dc1-b12d-b9de518aae7a",
+         "updatedAt": "2022-06-29T21:03:36.731Z"
+      }],
+      "endDate": "2023-01-01T00:00:00Z",
+      "pricingMethod": "FIXED_PRICE",
+      "type": "RESOURCE",
+      "fixedPrice": 500.0,
+      "createdAt": "2022-06-29T21:03:36.708Z",
+      "organization": {
+         "name": "subOrg",
+         "id": "e2c00fdb-d493-422c-889f-eb734806b80e"
+      },
+      "name": "fixed monthly price commitment",
+      "id": "aa7c81ab-478a-496b-a0ae-8f30459b0fa5",
+      "terminated": false,
+      "startDate": "2022-06-01T00:00:00Z",
+      "updatedAt": "2022-06-29T21:03:36.708Z"
+    }
+}
+
+```
+
+> Request body example for UTILITY_DISCOUNT commitments
+
+```json
+
+{
+   "committedProducts": [{
+      "product": {
+         "id": "73ebf9c8-ecf7-4ee6-b881-150023d5f442"
+      },
+      "committedAmount": 1000.0,
+      "discountAmount": 10
+   }],
+   "endDate": "2023-01-01T00:00:00Z",
+   "pricingMethod": "UTILITY_DISCOUNT",
+   "rateType": "FIXED_RATE",
+   "type": "RESOURCE",
+   "organization": {
+      "id": "e2c00fdb-d493-422c-889f-eb734806b80e"
+   },
+   "name": "fixed price commitment",
+   "startDate": "2022-06-01T00:00:00Z"
+}
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+   "data": {
+      "committedProducts": [{
+         "createdAt": "2022-06-29T21:06:55.163Z",
+         "product": {
+            "id": "73ebf9c8-ecf7-4ee6-b881-150023d5f442",
+            "sku": "f"
+         },
+         "discountAmount": 10,
+         "committedAmount": 1000.0,
+         "fixedRateReferencePrice": 10,
+         "id": "4a7be20e-23f2-4e83-883e-8e5c56e7552a",
+         "updatedAt": "2022-06-29T21:06:55.163Z"
+      }],
+      "endDate": "2023-01-01T00:00:00Z",
+      "pricingMethod": "UTILITY_DISCOUNT",
+      "type": "RESOURCE",
+      "rateType": "FIXED_RATE",
+      "createdAt": "2022-06-29T21:06:55.153Z",
+      "organization": {
+         "name": "subOrg",
+         "id": "e2c00fdb-d493-422c-889f-eb734806b80e"
+      },
+      "name": "fixed rate commitment",
+      "id": "7e2c15ae-c43b-44e9-9bbb-8e8821740f53",
+      "terminated": false,
+      "startDate": "2022-06-01T00:00:00Z",
+      "updatedAt": "2022-06-29T21:06:55.153Z"
+   }
+}
+
+```
+
+
+Attributes | &nbsp;
+---- | ---
+`name`<br/>*string* | The name of the commitment.
+`organization`<br/>*Organization* | The organization that owns the commitment.
```

Comment:
is this the org that owns the commitment or the org that this commitment is assigned to? should there be both?